clients/githubrepo/graphql.go

```diff
@@ -56,22 +56,32 @@ type graphqlData struct {
 		}
 		PullRequests struct {
 			Nodes []struct {
+				Author struct {
+					Login githubv4.String
+				}
 				Number      githubv4.Int
 				HeadRefOid  githubv4.String
 				MergeCommit struct {
-					AuthoredByCommitter githubv4.Boolean
+					Author struct {
+						User struct {
+							Login githubv4.String
+						}
+					}
 				}
 				MergedAt githubv4.DateTime
 				Labels   struct {
 					Nodes []struct {
 						Name githubv4.String
 					}
 				} `graphql:"labels(last: $labelsToAnalyze)"`
-				LatestReviews struct {
+				Reviews struct {
 					Nodes []struct {
+						Author struct {
```

Comment:
I think this can just be:

```
Author struct {
  Login githubv4.String
}
```

Reply:
I don't follow. The code your posted is the one I already have. Did you mean something different?

Reply:
I think you meant remove the Author field since we
re not using it anywhere. Removed.